res/css/structures/auth/_SetupEncryptionBody.scss

```diff
@@ -17,9 +17,11 @@ limitations under the License.
 .mx_SetupEncryptionBody_reset {
     color: $light-fg-color;
     margin-top: $font-14px;
```

Comment:
I guess the value for for font size is not supposed to be used as a margin value, btw.